R/pkg/R/mllib.R

```diff
@@ -171,7 +172,8 @@ predict_internal <- function(object, newData) {
 #' @note spark.glm since 2.0.0
 #' @seealso \link{glm}, \link{read.ml}
 setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
-          function(data, formula, family = gaussian, tol = 1e-6, maxIter = 25, weightCol = NULL) {
+          function(data, formula, family = gaussian, tol = 1e-6, regParam = 0.0, maxIter = 25,
+                   weightCol = NULL) {
```

Comment:
If say an R user call the function by `spark.glm(df, label ~ feature, gaussian, 1e-6, 25)`. This will break their code.